packages/router/src/__tests__/util.test.js

```diff
@@ -7,10 +7,26 @@ describe('matchPath', () => {
       params: { id: 7 },
     })
 
+    expect(matchPath('/post/{id:Int}', '/post/notAnInt')).toEqual({
```

Comment:
These should also fail for `Int`: 

* `'/post/2.0` 
* `'/post/.1` 
* `'/post/0.1` 
* `'/post/123abcd` 
* `'/post/abcd123`

Reply:
Indeed! Will add them

Reply:
👍 done!